frontend/beCompliant/src/pages/QuestionPage.tsx

```diff
@@ -31,69 +30,43 @@ export const QuestionPage = () => {
     isPending: commentsIsLoading,
   } = useFetchCommentsForQuestion(team ?? '', recordId);
 
-  const { mutate: submitAnswer } = useSubmitAnswers(team);
-
   if (questionIsLoading || answersIsLoading || commentsIsLoading) {
     return <LoadingState />;
   }
 
-  if (questionError || !question || answersError || commentsError) {
+  if (questionError || answersError || commentsError || !question || !answers) {
     return <ErrorState message="Noe gikk galt, prøv gjerne igjen" />;
   }
 
-  const handleSelectionAnswer = (e: React.ChangeEvent<HTMLInputElement>) => {
-    const newAnswer: string = e.target.value;
-    submitAnswer({
-      actor: 'Unknown',
-      recordId: recordId ?? '',
-      questionId: question.id,
-      question: question.question,
-      answer: newAnswer,
-      updated: '',
-      team: team,
-      answerType: question.metadata.answerMetadata.type,
-    });
-  };
-
-  const sikkerhetskontroller = question.metadata.optionalFields?.find(
-    (field) => field.key == 'Sikkerhetskontroller'
-  )?.value[0];
-
   return (
-    <Page gap="4" alignItems="center">
+    <Flex direction="column" marginTop="10">
+      <Button
+        variant="tertiary"
+        leftIcon="arrow_back"
+        colorScheme="blue"
+        alignSelf="start"
+        marginLeft="2"
+        as="a"
+        href={`/team/${team}`}
+      >
+        Tilbake
+      </Button>
       <Flex
+        alignSelf="center"
         flexDirection="column"
-        marginX="10"
         gap="2"
-        width={{ base: '100%', md: '50%' }}
+        width={{ base: '100%', md: '40%' }}
       >
-        <Heading textAlign="left" width="100%" marginBottom="30px">
-          Rediger
-        </Heading>
-        <Text fontSize="md">{question.id}</Text>
-        <Text fontSize="lg" as="b">
-          {sikkerhetskontroller}
-        </Text>
-        <Text fontSize="lg">
-          {'Sist endret ' + formatDateTime(question.updated ?? '')}
-        </Text>
-        <Text fontSize="lg" as="b">
-          Svar
-        </Text>
-        <RadioGroup name="Svar" defaultValue={answers.at(-1)?.answer}>
-          <Stack direction="column">
-            {question.metadata.answerMetadata.options?.map((option) => (
-              <Radio
-                key={option}
-                value={option}
-                onChange={handleSelectionAnswer}
-              >
-                {option}
-              </Radio>
-            ))}
-          </Stack>
-        </RadioGroup>
+        <QuestionDetails
+          question={question}
+          answerUpdated={answers.at(-1)?.updated.toString() ?? ''}
+        />
```

Comment:
Ref min kommentar på `QuestionDetails`

```suggestion
        <QuestionDetails
          question={question}
          answerUpdated={answers.at(-1)?.updated.toString() ?? ''}
          marginBottom='30'
        />
```

Reply:
Blir bare 30px når jeg setter inn 30, vet du hvorfor?